.travis.yml

```diff
@@ -50,8 +50,9 @@ script:
      stack --no-terminal test --ghc-options="$GHC_OPTIONS" $STACK_OPTS
    }
 
-   run_tests_using "solc-0.4.25"
+   #run_tests_using "solc-0.4.25"
```

Comment:
This PR looks good to me. The only concern is the time spent testing with 0.4.x, 0.5.x and 0.5.x. Should we keep support for the oldest version?

Reply:
Let's try to support it, if possible.